client/components/ResponsiveFlex/ResponsiveFlex.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import FormattedText from '../FormattedText';
+import Highlight from '../Highlight/Highlight';
+import ResponsiveFlex from './ResponsiveFlex';
+
+export default {
+  title: 'ResponsiveFlex',
+};
+
+export const Default = () => (
+  <>
+    <ResponsiveFlex breakpoint={1024}>
```

Comment:
Please simplify the example. There are a lot of elements here and it's not clear what should I be looking at. I propose we do just a very simple example which is enough to illustrate what the component is doing:

```
<ResponsiveFlex ...>
  <div> ... </div>
   <div> ... <div>
```

Inside the Divs you can add some text or images if you want, but it shouldn't be very complicated.